README.md

```diff
@@ -65,7 +69,45 @@ To configure the Welcome Bot, edit your `config.json` file with a message you wa
         },
 ```
 
-where
+This field can be JSON or a string containing JSON.
+
+### System Console
+
+In the UI, go to **System Console -> Plugins -> Welcome Bot** and edit the **Welcome Messages** text field to include the JSON array for messages you want to send. Use the following format:
+
+```
+[
+    {
+        "TeamName": "your-team-name, your-second-team-name",
+        "DelayInSeconds": 3,
+        "Message": [
+            "Your welcome message here. Each list item specifies one line in the message text."
+        ],
+        "AttachmentMessage": [
+            "Attachment message containing user actions"
+        ],
+        "Actions" : [
+            {
+                "ActionType": "button",
+                "ActionDisplayName": "User Action",
+                "ActionName": "action-name",
+                "ActionSuccessfulMessage": [
+                    "Message posted after the user takes this action and joins channels specified by 'ChannelsAddedTo'."
+                ],
+                "ChannelsAddedTo": ["channel-1", "channel-2"]
+            },
+            {
+                "ActionType": "automatic",
+                "ChannelsAddedTo": ["channel-3", "channel-4"]
+            }
+        ]
+    }
+]
+```
+
+If you see `[Object object]` in the text field, that's because the configuration was configured as JSON directly in your `config.json` instead of as a string.
```

Comment:
Can we instead JSON.stringify the data if it's not originally a string?

Reply:
@mickmister  Can you elaborate a bit on this, I am not really able to understand it.

Reply:
Regarding the other review comments, I am not able to push the code, as we don't have access to push the code. Will fix it afterwards

Reply:
I mean, if it's the case that the setting is an object (and not a JSON _string_), then the frontend can call `JSON.stringify` to make it into a string to put into the text box. This would require the plugin setting to be `custom` I think. Otherwise the admin cannot go and edit the current configuration if it just says `[Object object]`

Ideally we can just keep it as an object in the config file too, and not have a JSON string in the `config.json` file. Which would also require it to be a `custom` admin console setting